tutorials/power_distribution/api.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
```

Comment:
technically, I'm not sure if you need the pin_defs file since I don't think we're using them in this header